kit/src/main/java/com/oracle/javafx/scenebuilder/kit/skeleton/AbstractSkeletonCreator.java

```diff
@@ -38,13 +38,13 @@
 import java.util.Map;
 import java.util.ResourceBundle;
 
-abstract class AbstractSkeletonCreator {
+abstract class AbstractSkeletonCreator implements SkeletonConverter {
```

Comment:
Please change the license header from `Copyright (c) 2021,` to `Copyright (c) 2021, 2022,`.